xcodecommand/fallbackRunner.go

```diff
@@ -0,0 +1,48 @@
+package xcodecommand
+
+import (
+	"github.com/bitrise-io/go-utils/v2/command"
+	"github.com/bitrise-io/go-utils/v2/log"
+	version "github.com/hashicorp/go-version"
+)
+
+// FallbackRunner falls back to the raw xcodebuild runner in case xcbeautify (or other) is unavaialble
+type FallbackRunner struct {
+	runner         Runner
+	fallbackRunner Runner
+	logger         log.Logger
+}
+
+// NewFallbackRunner wraps an xcbeatufy (or other) runner
+func NewFallbackRunner(runner Runner, logger log.Logger, commandFactory command.Factory) *FallbackRunner {
+	return &FallbackRunner{
+		runner:         runner,
+		fallbackRunner: NewRawCommandRunner(logger, commandFactory),
```

Comment:
not sure if it make sense to create and store the fallbackrunner at this point, because if `CheckInstall` isn't failing, it never gets used.

What about creating it only when needed? (This way we would need to store the arguments needed for the NewRawCommandRunner)

Reply:
This file is now removed. Raw command runner is created on demand, in this other repo: https://github.com/bitrise-steplib/steps-xcode-test/pull/234.